### PR TITLE
26 privy transaction

### PR DESCRIPTION
**Pull request by bendsp**

### CODE REVIEW

**Comment by bendsp** [comment]
arbitrary messages can now be signed using Privy Provider. Will mostly be replaced by viem implementation #38

**Review by KentinPaille — APPROVED** [review]
ras

File: app/_layout.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ const Layout = () => {
           <Stack.Screen name="login" options={{ headerShown: false }} />
         </Stack>
       </ThemeWrapper>
+      <PrivyElements/>
```
Comment (PaulThblt) [inline app/_layout.tsx]: This is a requirement for privy related stuff right?
What is the difference between this and `PrivyProvider`?

**Review by PaulThblt — APPROVED** [review]

File: app/_layout.tsx (hunk 1 above)
Comment (bendsp) [reply app/_layout.tsx]: `PrivyProvider` is the wrapper that provides the application with use of all privy hooks. 
`PrivyElements` serves privy UI elements, which we currently do not use but may want to explore using as crutches. I am removing it for now.

**Pull request merged**